frontend/pages/Parts/[...deviceHandleItemType].tsx

```diff
@@ -75,6 +76,15 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
       };
    }
 
+   if (productList.deviceTitle && productList.deviceTitle !== deviceTitle) {
+      return {
+         redirect: {
+            permanent: true,
+            destination: `/Parts/${encodeDeviceTitle(productList.deviceTitle)}`,
```

Comment:
Might need to account for item type here too.

Reply:
https://github.com/iFixit/react-commerce/pull/556/commits/08d1ad5343fdbf38960a67d5dd3be934bf67d247